docs/catalog/cloud/evaluate-using-a-service-mesh.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
+- network
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Evaluate using a service mesh
```

Comment:
Pattern titles should be more action-oriented on a resource and any "evaluations" should be part of the assumptions or considerations sections below.